node/http/server.go

```diff
@@ -62,7 +62,7 @@ func New(listener net.Listener, metrics bool, database db.DB, version string,
 	mux.Handle("/grpc", grpcHandler)
 	if pprofEnable {
 		// Taken from https://artem.krylysov.com/blog/2017/03/13/profiling-and-optimizing-go-web-applications/
```

Comment:
Let's remove this comment now that we're changing the code. Almost identical lines are in the stdlib anyway.